package.json

```diff
@@ -70,20 +70,27 @@
   "homepage": "https://github.com/code-chunks/angular2-logger#readme",
   "dependencies": {},
   "devDependencies": {
-    "angular2": "^2.0.0-beta.17",
+    "@angular/common": "^2.0.0-rc.0",
+    "@angular/compiler": "^2.0.0-rc.0",
+    "@angular/core": "^2.0.0-rc.0",
+    "@angular/platform-browser": "^2.0.0-rc.0",
+    "@angular/platform-browser-dynamic": "^2.0.0-rc.0",
     "es6-promise": "^3.0.2",
     "es6-shim": "^0.35.0",
+    "zone.js": "^0.6.12",
     "reflect-metadata": "0.1.2",
     "rimraf": "^2.5.2",
-    "rxjs": "5.0.0-beta.6",
```

Comment:
I saw the issue and fixed it, see travis's result. Dependency issue was because I forgot to delete node_modules before doing npm i, so it was looking for beta.17 dependencies while also looking for rc0 ones, totally my bad.